contributing_to_docs/doc_guidelines.adoc

```diff
@@ -344,6 +442,18 @@ $ lorem.sh
 
 Without the line spaces the content is likely to be not parsed correctly.
 
+For long lines of code that you want to break up among multiple lines, use a backslash to show the line break. For example:
+
+----
+$ oc get endpoints --all-namespaces --template \
+    '{{ range .items }}{{ .metadata.namespace }}:{{ .metadata.name }} \
+    {{ range .subsets }}{{ range .addresses }}{{ .ip }} \
+    {{ end }}{{ end }}{{ "\n" }}{{ end }}' | awk '/ 172\.30\./ { print $1 }'
+----
+
```

Comment:
We should talk about the situation where there's a callout at the end of a line, and the syntax for that.